src/main/java/com/marklogic/mule/extension/Operations.java

```diff
@@ -125,21 +127,23 @@ public PagingProvider<DatabaseClient, Result<InputStream, DocumentAttributes>> s
         @DisplayName("Directory") @Optional @Example("/customerData") String directory,
         @DisplayName("REST Transform") @Optional String restTransform,
         @DisplayName("REST Transform Parameters") @Optional String restTransformParameters,
-        @DisplayName("REST Transform Parameters Delimiter") @Optional @Example(",") String restTransformParametersDelimiter
+        @DisplayName("REST Transform Parameters Delimiter") @Optional @Example(",") String restTransformParametersDelimiter,
+        @DisplayName("Use Consistent Timestamp") @Optional(defaultValue = "True") boolean useConsistentTimestamp
     ) {
         final int finalPageLength = pageLength != null ? pageLength : 100;
 
         return new PagingProvider<DatabaseClient, Result<InputStream, DocumentAttributes>>() {
             int totalDocumentsDelivered = 0;
             Integer currentPage = -1;
+            Long searchTimeStamp = -1L;
```

Comment:
Do `serverTimestamp` as that's the variable used in other projects, including the Java Client.